src/_tests/cachedQueries.json

```diff
@@ -0,0 +1,271 @@
+{
+  "getProjectBoardColumns": [
```

Comment:
Generated by `npm run update-test-data`. Currently exists so tests can run without an auth token, but we could use this info in production if needed.